CONTRIBUTING.md

```diff
@@ -10,6 +10,7 @@
 * Code the algorithm/DS following the styleguide defined below.
 * Send a PR. 
 * Be sure to not include any compiled binaries in the patch.
+* While sending a PR make sure you follow one PR per issue rule.
```

Comment:
Yes and practice of squashing is very much important in the workflow and it is good if u learn it as early as possible. doesn't it @aviaryan  @singhpratyush